CHANGELOG.md

```diff
@@ -2,6 +2,81 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [2.0.0] - 2019-07-??
+ - New Features
+   - `pysatData` directory created in user's home directory if no directory specified
+   - Added preliminary support for `xarray` to the `instrument` object
+   - Support for `today`, `tomorrow`, and `yesterday` as datetime objects
+   - Added `model_utils`, featuring preliminary support for data-model comparison
+   - Added support for 1d median in seasonal averages
+   - Added routine to convert from kp to Ap
+   - Added `pyglow` integration support for python 3.x
```

Comment:
Added option to check that loaded data has a unique and monotonic time index. Will be enforced in a future version.

Reply:
Refactored data access through the Instrument object and expanded testing. (may be technically redundant with adding xarray)

Reply:
Added .empty attribute to Instrument object, True when no data loaded.

Reply:
Added .index access mechanism to Instrument object, providing consistent access to the pandas DatetimeIndex associated with loaded data.

Reply:
Added mechanism to return a list of loaded variables, .variables.

Reply:
Added Instrument method to concat input data with data already loaded into Instrument object.

Reply:
Updated format of printed dates to day month name and year, 01 January 2001.

Reply:
Added Instrument property .date, returns date of loaded data.

Reply:
Added download_updated_files, Instrument method that downloads any remote data not currently on the local machine.

Reply:
Added remote_date_range, an Instrument method that returns first and last date for remote data.

Reply:
Download method now defaults to most recent data (near now).

Reply:
updated.